jekyll/_cci2/server-3-install.adoc

```diff
@@ -143,6 +143,41 @@ The domain you selected for your CircleCI installation is the Homepage URL and *
 
 NOTE: If using GitHub Enterprise, you will also need a personal access token and the domain name of your GitHub Enterprise instance. You must also enable HTTP API Rate Limiting from the management console.
 
+### Vault
+*Internal* will deploy the default Vault instance inside the CircleCI K8s namespace.  The application will be automatically configured.
+*External* will not install the default Vault instance with the CircleCI application.  Select this option if using a Vault instance ou.  Will require the user to configure the following settings:
+
+. URL: ex. `http://vault:8200`
+. Transit Path: The name of the secrets engine.  The default is `transit`. See the https://www.vaultproject.io/docs/secrets/transit#setup[Vault documentation] for more details.
```

Comment:
Nit: "name" feels a little ambiguous in that it doesn't really match anything in the Hashicorp docs

```suggestion
. Transit Path: The path of the transit secrets engine.  The default is `transit`. See the https://www.vaultproject.io/docs/secrets/transit#setup[Vault documentation] for more details.
```

Reply:
Fixed, thanks!